### PR TITLE
Add a new blog post presenting raw image streaming

### PR DESCRIPTION
**Pull request by maelk**
This blog post details a new feature that was recently added in
Metal3, and how to leverage it for memory constrained deployments.

### CODE REVIEW

File: _posts/2020-07-05-raw-image-streaming.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+```bash
+qemu-img convert -O raw "${IMAGE_NAME}" "${IMAGE_RAW_NAME}"
+```
+
+Once converted the image format needs to be provided to Ironic through the
+BareMetalHost (BMH) image spec field. If not provided, Ironic will assume that
+the format is unspecified and download it in memory first.
+
+The following is an example of the BMH image spec field in Metal3-dev-env.
```
Comment (furkatgofurov7) [inline _posts/2020-07-05-raw-image-streaming.md:70]: it should be **m**etal3-dev-env in my opinion to maintain consistency

File: _posts/2020-07-05-raw-image-streaming.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
+---
+title: "Raw image streaming available in Metal3"
+date: 2020-07-05
+draft: false
+categories: ["metal3", "baremetal", "raw image", "image streaming"]
+author: Maël Kimmerlin
+---
+
+Metal3 supports multiple types of images for deployment, the most
+popular being QCOW2. We have recently added support for a feature of Ironic
```
Comment (furkatgofurov7) [inline _posts/2020-07-05-raw-image-streaming.md:10]: should we add `QCOW2` in yaspeller.json as well?

File: _posts/2020-07-05-raw-image-streaming.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+## What is raw image streaming ?
+
+The target image format when writing to disk is raw. That's why the images in
+formats like QCOW2 must be processed before being written to disk. However, if
+the image that is downloaded is already in raw format, then no processing is
+needed.
+
+Ironic leverages this, and instead of first downloading the image and then
+processing it before writing it to disk, it will directly write the
+downloading file to the disk. This feature is known as image streaming.
```
Comment (furkatgofurov7) [inline _posts/2020-07-05-raw-image-streaming.md:40]: can you please specify what kind of file is it? an image?

File: _posts/2020-07-05-raw-image-streaming.md (hunk 2 above)
Comment (maelk) [reply _posts/2020-07-05-raw-image-streaming.md:10]: the spellcheck did not fail on it, so I guess it is not required

File: _posts/2020-07-05-raw-image-streaming.md (hunk 1 above)
Comment (maelk) [reply _posts/2020-07-05-raw-image-streaming.md:70]: or maybe Metal3 Dev Env ?

**Comment by smoshiur1237** [comment]
Brief description on raw image streaming with examples. Good writing, understandable and well done.

**Comment by dhellmann** [comment]
Nice!

/approve
/lgtm

**Pull request merged**